src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.h

```diff
@@ -2,20 +2,84 @@
 #define Z_EN_S_GORO_H
 
 #include "global.h"
+#include "objects/object_oF1d_map/object_oF1d_map.h"
 
-#define EN_S_GORO_GET_PARAM_F(thisx) ((thisx)->params & 0xF)
+#define EN_S_GORO_GET_MAIN_TYPE(thisx) ((thisx)->params & 0xF)
+#define EN_S_GORO_SCENEFLAG_INDEX(thisx) (((thisx)->params & 0x7F0) >> 4)
+
+#define EN_S_GORO_ACTIONFLAG_ROLLEDUP (1 << 0)
+#define EN_S_GORO_ACTIONFLAG_FACEPLAYER (1 << 1)
+#define EN_S_GORO_ACTIONFLAG_EYESOPEN (1 << 2)
+#define EN_S_GORO_ACTIONFLAG_EARSCOVERED (1 << 3)
+#define EN_S_GORO_ACTIONFLAG_ENGAGED (1 << 4)
+#define EN_S_GORO_ACTIONFLAG_LASTMESSAGE (1 << 5)
+#define EN_S_GORO_ACTIONFLAG_GKQUIET_ACKNOWLEDGED (1 << 6)
+#define EN_S_GORO_ACTIONFLAG_SNOREPHASE (1 << 7)
+#define EN_S_GORO_ACTIONFLAG_UNK0100 (1 << 8)
+#define EN_S_GORO_ACTIONFLAG_HANDTAP (1 << 9)
+#define EN_S_GORO_ACTIONFLAG_TIRED (1 << 10)
+#define EN_S_GORO_ACTIONFLAG_SUPPRESS_SNORE (1 << 11)
+
+#define EN_S_GORO_BOMBBUYFLAG_TALKED_HUMAN (1 << 0)
+#define EN_S_GORO_BOMBBUYFLAG_TALKED_GORONPK (1 << 1)
+#define EN_S_GORO_BOMBBUYFLAG_TALKED_DEKU (1 << 2)
```

Comment:
Are either the `ACTIONFLAG`s or `BOMBBUYFLAG`s used outside `z_en_s_goro.c`? If not, these can likely be moved to the `.c` file at the top, as these seem internal to the system